python/sphinx_docs/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "opm-simulators-docs"
+version = "0.1.0"
```

Comment:
not sure if this needs to be in sync?

Reply:
I think not? Since it is not going to be published. The python project is only used to build the sphinx documentation.

Reply:
yeah i kinda guessed, just wasn't sure if something would get their panties in a bundle or not.